src/detect-flowbits.c

```diff
@@ -105,6 +142,15 @@ static int DetectFlowbitMatchIsset (Packet *p, const DetectFlowbitsData *fd)
 {
     if (p->flow == NULL)
         return 0;
+    if (fd->or_list_size > 0) {
+        int result = 0;
+        for (int i = 0; i < fd->or_list_size; i++) {
```

Comment:
`i` is signed, `fd->or_list_size` unsigned. I suspect we may see warnings for that depending on compiler flags.